### PR TITLE
hide deprecation warning

### PR DESCRIPTION
**Pull request by adeptofvoltron**
with symfony 6.1 I got such warning:

```
2023-03-28T12:23:12+02:00 [info] User Deprecated: Method "Symfony\Component\HttpKernel\Bundle\Bundle::build()" might add "void" as a native return type declaration in the future. Do the same in child class "Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle" now to avoid errors or add an explicit @return annotation to suppress this message.
```

such change repair it

### CODE REVIEW

File: DoctrineMigrationsBundle.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
  */
 class DoctrineMigrationsBundle extends Bundle
 {
-    public function build(ContainerBuilder $container)
+    public function build(ContainerBuilder $container): void
```
Comment (greg0ire) [inline DoctrineMigrationsBundle.php:17]: The class is not final, so this is technically a breaking change. You should use a phpdoc annotation instead:

```suggestion
    /** @return void */
    public function build(ContainerBuilder $container)
```

Comment (stof) [reply DoctrineMigrationsBundle.php:17]: I also suggest adding `@final since 3.3.0` (or `3.4.0` depending on what is the next minor version) on that class

Comment (greg0ire) [reply DoctrineMigrationsBundle.php:17]: The next minor version should be 3.3.0.

**Review by derrabus — CHANGES_REQUESTED** [review]
As @greg0ire said, we cannot add a return type to a non-final method in a minor release.

**Comment by nicolas-grekas** [comment]
@adeptofvoltron up to do the suggested changes?

**Comment by mickverm** [comment]
https://github.com/doctrine/DoctrineMigrationsBundle/pull/494 has been merged.

**Comment by derrabus** [comment]
Thanks. Let's close this one then.

**Pull request closed without merging**